### PR TITLE
Tools UI

### PR DESCRIPTION
**Pull request by ngd-adams**
changed the risk survey UI, and results UI

### CODE REVIEW

File: app/globals.css [hunk 1]
```diff
@@ -11,4 +11,8 @@ body {
   .text-balance {
     text-wrap: balance;
   }
+}
+
+:root {
+  --gradient: linear-gradient(90deg, rgba(111,84,162,1) 0%, rgba(178,118,203,1) 50%, rgba(220,42,145,1) 100%);
```
Comment (jskerman) [inline app/globals.css:17]: This is certainly a way to do this, there are "nicer" ways but this is fine

File: app/signup/page.tsx [hunk 2]
```diff
@@ -1,7 +1,64 @@
...
+        </Stack>
+        <Center>
+          <AuthCta />
+        </Center>
+      </Card>
+    </Flex>
+  );
 };
 
+const AuthCta = () => (
```
Comment (jskerman) [inline app/signup/page.tsx:53]: What is this component doing?

File: components/ui/header.tsx [hunk 3]
```diff
@@ -60,23 +63,49 @@ const Nav = () => (
...
     <Flex gap={4}>
       <Link href={"/courses"}>
-        <Button colorScheme="hollywood" shadow="5px 5px 0 black">Courses</Button>
+        <Button colorScheme="hollywood" shadow="5px 5px 0 black">
+          Courses
+        </Button>
       </Link>
       <Link href={"/tools"}>
-        <Button colorScheme="hollywood" shadow="5px 5px 0 black">Tools</Button>
+        <Button colorScheme="hollywood" shadow="5px 5px 0 black">
```
Comment (jskerman) [inline components/ui/header.tsx:71]: Not sure is `shadow` props is doing anything here. Doesn't a button already have this style applied in `theme.ts`

File: theme.ts [hunk 4]
```diff
@@ -56,15 +56,24 @@ export const colors = {
     "300": "#92095c",
     "400": "#046aa0",
   },
-  gradient: {
-    "400": "linear-gradient(90deg, rgba(111,84,162,1) 0%, rgba(178,118,203,1) 50%, rgba(220,42,145,1) 100%)",
+
+  glowstone: {
```
Comment (jskerman) [inline theme.ts:60]: love it

File: theme.ts [hunk 5]
```diff
@@ -56,15 +56,24 @@ export const colors = {
...
+    "700": "#c07a2a",
+    "800": "#97611f",
   },
+
 } as const;
 
 const components = {
   Button: {
     baseStyle: {
-      // shadow: "5px 5px 0 black",
```
Comment (jskerman) [inline theme.ts:67]: Ah I see, you removed it!

File: app/signup/page.tsx (hunk 2 above)
Comment (ngd-adams) [reply app/signup/page.tsx:53]: Ideally this is just a link to the google auth but then doesnt change into the account icon

**Comment by jskerman** [comment]
LGTM! Approved, merge whenever

File: components/ui/header.tsx (hunk 3 above)
Comment (ngd-adams) [reply components/ui/header.tsx:71]: yeah it is the base button theme I removed the shadow

File: app/signup/page.tsx (hunk 2 above)
Comment (jskerman) [reply app/signup/page.tsx:53]: Adding px={4} around an AuthButton turns it into a AuthCta then? flexDir and gap aren't doing anything here unless AuthButton root is a react fragment.

**Pull request merged**